app/src/pages/playground/InvocationParametersForm.tsx

```diff
@@ -275,7 +276,7 @@ export const InvocationParametersForm = ({
           updateInstanceModelInvocationParameters({
             instanceId: instance.id,
             invocationParameters: instance.model.invocationParameters.map(
-              (p) => (p.invocationName === field.invocationName ? input : p)
+              (p) => (areInvocationParamsEqual(p, field) ? input : p)
```

Comment:
@cephalization we might need to rethink the effect right above this, i think it should have caused a refresh of the form but did not when the stop parameter got its name updated. which i think it should have rerendered the form with the correct value at that point?

Reply:
Is this still accurate or was this a symptom of the span.invocation_params incorrectly falling back to openai?

Reply:
yeah so i think we still want this everywhere, i don't think it will ever lead to anything wrongly being picked up and it increases surface area of picking up the right things